functions/payment.js

```diff
@@ -1,282 +0,0 @@
-const admin = require("firebase-admin");
-const functions = require("firebase-functions");
-
-const PROJECT_ID = functions.config().project.id;
-const STRIPE_CONFIG = functions.config().stripe;
-
-// Set your secret key. Remember to switch to your live secret key in production!
-// See your keys here: https://dashboard.stripe.com/account/apikeys
-const stripe = require("stripe")(STRIPE_CONFIG.secret_key);
-
-const PURCHASE_TABLE = "purchases";
-const CUSTOMER_TABLE = "customers";
-
-const authenticationCheck = (context) => {
-  if (!context.auth || !context.auth.token) {
-    throw new functions.https.HttpsError("unauthenticated", "Please log in");
-  }
-
-  if (context.auth.token.aud !== PROJECT_ID) {
-    throw new functions.https.HttpsError("permission-denied", "Token invalid");
-  }
-};
-
-const getEventPrice = async (venueId, eventId) => {
-  const event = (
-    await admin
-      .firestore()
-      .collection("venues")
-      .doc(venueId)
-      .collection("events")
-      .doc(eventId)
-      .get()
-  ).data();
-  return event.price;
-};
```

Comment:
Have we removed `event.price` from `VenueEvent` in `src/types/venues.ts`? And possibly also `collective_price` by the looks of things there?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40